fdbserver/DDRelocationQueue.actor.cpp

```diff
@@ -1423,6 +1423,8 @@ ACTOR Future<Void> dataDistributionRelocator(DDQueue* self,
 	state double startTime = now();
 	state std::vector<UID> destIds;
 	state uint64_t debugID = deterministicRandom()->randomUInt64();
+	state bool enablePhysicalShardMove =
```

Comment:
nit: maybe `enableShardMove`? Since currently the move is still via key-value, not really physical.

Reply:
Done.